spring-integration-core/src/main/java/org/springframework/integration/dispatcher/UnicastingDispatcher.java

```diff
@@ -137,18 +137,24 @@ private boolean doDispatch(Message<?> message) {
 		if (!handlerIterator.hasNext()) {
 			throw new MessageDispatchingException(message, "Dispatcher has no subscribers");
 		}
-		List<RuntimeException> exceptions = new ArrayList<RuntimeException>();
+		List<RuntimeException> exceptions = new ArrayList<>();
 		while (!success && handlerIterator.hasNext()) {
 			MessageHandler handler = handlerIterator.next();
 			try {
 				handler.handleMessage(message);
 				success = true; // we have a winner.
 			}
-			catch (Exception e) {
-				RuntimeException runtimeException = IntegrationUtils.wrapInDeliveryExceptionIfNecessary(message,
-						() -> "Dispatcher failed to deliver Message", e);
+			catch (Exception ex) {
+				RuntimeException runtimeException =
+						IntegrationUtils.wrapInDeliveryExceptionIfNecessary(message,
+								() -> "Dispatcher failed to deliver Message", ex);
```

Comment:
Why this line wrap? It was previously 112.

Reply:
Well, my own preferences: I find it is much readable when variable initialization is multi-line. So, I start initialization block fully from a new line.

Reply:
OK